README.md

```diff
@@ -72,6 +72,8 @@ This project is a web page that lets user list To Do tasks. Built using HTML, CS
 
 - **Homepage**
 - **Todo Tasks section**
+- **Add and Remove Tasks**
+- **Manage tasks with local storage**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
- [ ] If you have not deployed the project yet, I think it would be better to indicate that the `live demo` link is not available. Having broken links on your readme file hinders user experience: If users encounter broken links in the README file, they may become frustrated and give up on using the software. This can harm the adoption of the project and lead to a decrease in its popularity.